### PR TITLE
docs: wifi: Update release notes

### PR DESCRIPTION
**Pull request by krish2718**
Update release notes with Wi-Fi protocol, sample and documentation
additions.

Signed-off-by: Krishna T <krishna.t@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -363,6 +370,10 @@ Zigbee samples
   * Added support for :ref:`zephyr:nrf52840dongle_nrf52840`.
   * Added an option to build :ref:`zigbee_shell_sample` sample with the nRF USB CDC ACM as shell backend.
 
+Wi-Fi samples
+-------------
+
+* Added a :ref:`wifi_shell_sample` sample with shell support
```
Comment (richabp) [inline doc/nrf/releases/release-notes-changelog.rst:376]:
```suggestion
* Added :ref:`wifi_shell_sample` sample with the shell support.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -649,6 +660,7 @@ Documentation
   * Added software maturity entries for security features: TF-M, PSA crypto, Immutable bootloader, HW unique key.
   * A section about NFC in the :ref:`app_memory` page.
   * A note in the :ref:`ug_ble_controller` about the usage of the Zephyr LE Controller.
+  * Documentation for the nrf700x PDK
```
Comment (richabp) [inline doc/nrf/releases/release-notes-changelog.rst:663]:
```suggestion
  * Documentation for :ref:`ug_nrf70`.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -96,6 +96,13 @@ ESB
 
 * Fixed the ``update_radio_crc()`` function in order to correctly configure the CRC's registers (8 bits, 16 bits, or none).
 
+Wi-Fi
```
Comment (richabp) [inline doc/nrf/releases/release-notes-changelog.rst:99]: Not sure if we should add this here as no Protocol has been added.

Comment (krish2718) [reply doc/nrf/releases/release-notes-changelog.rst:99]: nrf7002 introduces new Wi-Fi protocol that can be used by applications.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -96,6 +96,13 @@ ESB
...
+-----
+
+* Added support for nrf700x PDK that supports Wi-Fi SoC
+
+See `Wi-Fi samples`_ for details about how to use Wi-Fi in tour application.
```
Comment (shantha-14) [inline doc/nrf/releases/release-notes-changelog.rst:104]: typo ? "tour"

File: doc/nrf/releases/release-notes-changelog.rst [hunk 5]
```diff
@@ -96,6 +96,13 @@ ESB
 
 * Fixed the ``update_radio_crc()`` function in order to correctly configure the CRC's registers (8 bits, 16 bits, or none).
 
+Wi-Fi
+-----
+
+* Added support for nrf700x PDK that supports Wi-Fi SoC, which implements IEEE 802.11 protocols.
```
Comment (richabp) [inline doc/nrf/releases/release-notes-changelog.rst:102]:
```suggestion
* Added support for nRF700x DK that supports Wi-Fi SoC, which implements IEEE 802.11 protocols.
```

**Review by richabp — COMMENTED** [review]
Added a small suggestion for consistency.

**Review by shantha-14 — APPROVED** [review]

**Pull request merged**